ui/plugins/controller.js

```diff
@@ -243,6 +243,9 @@ treeherder.controller('PluginCtrl', [
                         $scope.logParseStatus = $scope.job_log_urls[0].parse_status;
                     }
 
+                    // Provide a parse status for the model
+                    $scope.jobLogsAllParsed = $scope.job_log_urls ? $scope.job_log_urls.every(jlu => (jlu.parse_status !== 'pending')) : false;
```

Comment:
If I were being picky, I would write this as something more like

```
$scope.jobLogsAllParsed = ($scope.job_log_urls ?
                                             $scope.job_log_urls.every(jlu => jlu.parse_status !== 'pending') :
                                             false);
```
i.e. shorter lines somehow and no extra parens in the function expression. But I'm not insisting.